E0284: type annotations required

Running the example in the README on both 1.23.0 and 1.25.0-nightly produces:

```
error[E0284]: type annotations required: cannot resolve `<ethereum_types::U256 as std::ops::Mul<_>>::Output == ethereum_types::U256`
 --> src/main.rs:7:13
   |
 7 |         val = val * 2.into()
   |                   ^
```

--- a/README.md
+++ b/README.md
@@ -19,7 +19,7 @@ use ethereum_types::U256;
 
 fn main() {
 	let mut val: U256 = 1023.into();
-	for _ in 0..200 { val = val * 2.into() }
+	for _ in 0..200 { val = val * 2u32 }
 	assert_eq!(
 		&format!("{}", val),
 		"1643897619276947051879427220465009342380213662639797070513307648"

